docs/build/smart-contracts/getting-started/hello-world-frontend.mdx

```diff
@@ -0,0 +1,130 @@
+---
+sidebar_position: 50
+sidebar_label: "5. Build a Hello World Frontend"
+title: "Build a frontend for the Hello World contract"
+description: "Build a frontend for the Hello World contract by using Stellar CLI to generate TypeScript bindings."
+---
+
+# 5. Build a Hello World Frontend
+
+In the previous examples, we invoked the contracts using the Stellar CLI, and in this last part of the guide we'll create a web app that interacts with the Hello World contract through TypeScript bindings.
+
+:::info
+
+This example shows one way of creating a binding between a contract and a frontend, for a more comprehensive guide see the [Build a Dapp Frontend](../../apps/dapp-frontend.mdx) documentation.
+
+:::
+
+## Initialize a frontend toolchain
+
+You can build a Soroban app with any frontend toolchain or integrate it into any existing full-stack app. For this tutorial, we're going to use [Astro](https://astro.build/). Astro works with React, Vue, Svelte, any other UI library, or no UI library at all. In this tutorial, we're not using a UI library. The Soroban-specific parts of this tutorial will be similar no matter what frontend toolchain you use.
+
+If you're new to frontend, don't worry. We won't go too deep. But it will be useful for you to see and experience the frontend development process used by Soroban apps. We'll cover the relevant bits of JavaScript and Astro, but teaching all of frontend development and Astro is beyond the scope of this tutorial.
+
+Let's get started.
+
+You're going to need [Node.js](https://nodejs.org/en/download/package-manager/) v18.14.1 or greater. If you haven't yet, install it now.
```

Comment:
We probably want to recommend node v20 or greater. v18 is set to reach EOL next month

Reply:
Done